issuer-list/src/main/java/com/adyen/checkout/issuerlist/IssuerListSpinnerView.kt

```diff
@@ -38,32 +42,40 @@ abstract class IssuerListSpinnerView<
         IssuerListComponentT>(context, attrs, defStyleAttr),
     AdapterView.OnItemSelectedListener {
 
-    private lateinit var issuersSpinner: AppCompatSpinner
-    private lateinit var issuersAdapter: IssuerListSpinnerAdapter
+    protected val termsAndConditionsTextView: TextView?
+        get() = rootView.findViewById<TextView>(R.id.textview_termsAndConditions)
+    private val autoCompleteTextViewIssuers: AutoCompleteTextView
+        get() = rootView.findViewById(R.id.autoCompleteTextView_issuers)
+    private val textInputLayoutIssuers: TextInputLayout
+        get() = rootView.findViewById(R.id.textInputLayout_issuers)
+
+    private val issuersAdapter: SimpleTextListAdapter<IssuerModel> = SimpleTextListAdapter(context)
```

Comment:
Why did we change this from `IssuerListSpinnerAdapter` to `SimpleTextListAdapter`? `IssuerListSpinnerAdapter` items has an image and `SimpleTextListAdapter` items don't. And it would break payment methods like `iDeal`. Did you test that?

Reply:
I will revert all changes in `IssuerListSpinner` and will create a new `OnlineBankingSpinnerView` as yeah maybe this will break stuff.